### ISSUE TITLE
[ui-material] Input type "file" isn't supported by matInput.

### ISSUE THREAD

**Issue by Sntax** [udos86/ng-dynamic-forms#671]
## I'm submitting a

<pre><code>
[x] Bug / Regression
[ ] Feature Request / Proposal
[ ] Question
</code></pre>

## I'm using

<pre><code>
NG Dynamic Forms Version: 5.4.4

[ ] Basic UI
[ ] Bootstrap UI  
[ ] Foundation UI
[ ] Ionic UI
[ ] Kendo UI
[x] Material  
[ ] NG Bootstrap
[ ] Prime NG
</code></pre>

## Description

According to the documentation, file-upload controls should be available for the ui-material package by _using a DynamicInputModel with inputType: "file"._ Unfortunately I am met with the following error when attempting to set an `inputType: 'file',` on the form models:

> ERROR Error: Input type "file" isn't supported by matInput.

![screen shot 2018-01-29 at 13 44 23](https://user-images.githubusercontent.com/3358672/35508853-971f7fae-04fa-11e8-94ef-634def494c3d.png)

My model that is generating this error looks like this:

```
new DynamicInputModel({
  label: name,
  inputType: 'file',
  placeholder: name,
  id: `${item.id}_content`,
});
```

There also appears to be no example of file-upload functionality on the DEMO for ui-material.

**Comment by udos86** [comment]
@Sntax 

closed as duplicate of #611

**Comment by Sntax** [comment]
@udos86 - Thanks for the heads up. It would probably be worthwhile to update the documentation to correctly display that this is not yet a supported feature 😃  .

Thanks for the library so far. Great work.
